migrations/011.do.daily-stations-idx-day.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX daily_stations_day ON daily_stations (day);
```

Comment:
Oops, I should have added `CONCURRENTLY` to let Postgres build the index in the background and don't block the startup of the spark-evaluate service 🙈

Reply:
The index is live, the migration finished in ~29 second 😅 

```
2024-07-03T12:22:23Z app[e2867541be3e68] cdg [info]Migrating DB schema from version 10 to version 11
2024-07-03T12:22:52Z app[e2867541be3e68] cdg [info]Migrated DB schema to version 11
```